cairo/src/matrices.rs

```diff
@@ -65,6 +65,43 @@ impl Matrix {
         }
     }
 
+    pub fn xx(self) -> c_double {
+        self.xx
+    }
+    pub fn set_xx(&mut self, xx: f64) {
+        self.xx = xx;
+    }
+    pub fn yx(self) -> c_double {
```

Comment:
```suggestion
    pub fn yx(&self) -> c_double {
```

would make more sense. It's not super-cheap

Reply:
Should we remove the `Copy` impl then?